app/forms/idv/api_document_verification_form.rb

```diff
@@ -95,9 +95,13 @@ def throttle_if_rate_limited
 
     def throttled_else_increment
       return unless document_capture_session
-      @throttled = Throttler::IsThrottledElseIncrement.call(
-        document_capture_session.user_id,
-        :idv_acuant,
+      @throttled = throttler.throttled_else_increment?
+    end
+
+    def throttler
+      Throttle.for(
```

Comment:
Seems like memoization would be helpful here, since `submit` calls both `throttled_else_increment` and `remaining_attempts`.

Reply:
yup, fixed in a07b5cc86fe3facdb8d814aff5079f6a939e6c27